client/my-sites/customer-home/main.jsx

```diff
@@ -159,7 +155,7 @@ const Home = ( {
 	const headerActions = (
 		<>
 			<Button href={ site.URL } onClick={ trackViewSiteAction } target="_blank">
-				{ isGlobalSiteViewEnabled ? translate( 'View site' ) : translate( 'Visit site' ) }
```

Comment:
After the `layout/dotcom-nav-redesign-v2` flag was dropped from `getIsGlobalSiteViewEnabled()`, this logic does not make sense anymore, so this PR removes it and just use `View site`.